cli/src/diff_util.rs

```diff
@@ -92,7 +92,7 @@ pub struct DiffFormatArgs {
     #[arg(long, short)]
     pub summary: bool,
     /// Show a histogram of the changes
-    #[arg(long)]
+    #[arg(long, short_alias = 'S')]
```

Comment:
I think this should be `short` instead of `short_alias` (it's strange to define an alias for something that doesn't exist, which is what I think this does).

Reply:
done